pkg/config/config.go

```diff
@@ -238,6 +238,8 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("remote_configuration.config_root", "")
 	config.BindEnvAndSetDefault("remote_configuration.director_root", "")
 	config.BindEnvAndSetDefault("remote_configuration.refresh_interval", 1*time.Minute)
+	config.BindEnvAndSetDefault("remote_configuration.tracer_cache.size", 0)
+	config.BindEnvAndSetDefault("remote_configuration.tracer_cache.ttl_seconds", 0)
```

Comment:
There is more logic on defaulting this in the constructor: https://github.com/DataDog/datadog-agent/pull/9939/files#diff-78c027ff44e6b433b5ef173cd2a44b503ebeeb7fbb457f5e92832a1edae27d89R494-R508
This pattern was there before, so I went with it.

Reply:
besides consistency with some keys, this is only increasing a misconfiguration in case of code changes. Not all configuration entry use default constructor values to check if the field is set. 
I would also suggest to put the defaults here and let the config struct deal with it

Reply:
Yeah, I can put sane defaults here. I would still leave the logic in the constructor, because values `<=0` for ttl and size are not permissible.